dev/puppetserver.conf

```diff
@@ -3,7 +3,7 @@ global: {
 }
 
 product: {
-    update-server-url: "http://localhost/"
+    update-server-url: "http://some.update.server.that.does.not.exist/"
```

Comment:
Mostly what I was going for here is for the error message to contain something that made it obvious what the stack trace was about.